intents-resolver-ui/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "intents-resolver",
+  "version": "0.1.0",
+  "private": true,
+  "homepage": ".",
+  "dependencies": {
+    "@glue42/desktop": "^5.18.2",
+    "@glue42/theme": "^4.0.5",
```

Comment:
This should also be from the @interopio scope - I'm not sure which one of the two available theme packages.